contrib/oadp-recovery/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM registry.ci.openshift.org/openshift/release:rhel-9-release-golang-1.24-openshift-4.21 AS builder
+WORKDIR /workspace
+
+ENV GOFLAGS=-mod=mod
+
+# Copy go.mod and go.sum
+COPY go.mod go.sum ./
+RUN go mod download
```

Comment:
This `go mod download` could probably be run after `COPY . ./` which would not need copying go.mod and go.sum explicitly - they would be already copied.